### ISSUE TITLE
FNX-12534 ⁃ [Telemetry] Autoplay

### ISSUE THREAD

**Issue by vesta0** [mozilla-mobile/fenix#11579]
## Description & Product Manager / Data Scientist User Story
As a product manager, I want to know if users change their autoplay setting, so I can better understand whether this feature or its specific options are relevant and useful to our users or whether this feature is discoverable. 

## What questions will you answer with this data?
-What percentage of users open Site permissions => Autoplay option under settings
-What percentage of users change their autoplay default

## Acceptance Criteria
* [ ] ENG files a [DS JIRA](https://jira.mozilla.com/projects/DO/issues/DO-228?filter=allopenissues) request outlining their methodology.
* [ ] DS sign off on instrumentation methodology addressing product questions.
* [ ] Event pings can be queried via re:dash
* [ ] Event pings can be queried via amplitude
* [ ] We are sending telemetry events for the actions listed in the requirements
* [ ] We have documented the telemetry
* [ ] We have asked a data steward to [review](https://github.com/mozilla/data-review/blob/master/request.md) the telemetry

**Comment by sblatz** [comment]
Let's add autoplay toggle to the `preferences` metrics ping.

**Comment by sblatz** [comment]
@vesta0 can we handle how users flow through the settings fragment screens as a larger ticket? This would answer: `What percentage of users open Site permissions => Autoplay option under settings`? We can also do it on this ticket if it's critical :)

**Comment by vesta0** [comment]
@sblatz yes we can we handle how users flow through the settings fragment screens in a separate ticket as it is not critical.

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on **`Nightly Build 200730`** using a Google Pixel 3a (Android 10)

✔️ A user visited the autoplay settings screen - Ping **543ae510-212d-44a5-95e0-c11cd143f914**
```
"events": [
        {
          "category": "autoplay",
          "name": "visited_setting"
```
✔️ A user changed their autoplay setting to block audio and video on cellular data only - Ping **64f90062-2181-47a4-9223-1feb7e556ac9**
```
"events": [
        {
          "category": "autoplay",
          "extra": {
            "autoplay_setting": "block_cellular"
          },
          "name": "setting_changed"
```

✔️ A user changed their autoplay setting to block audio only - Ping **75daf84d-b455-47f0-893f-c9ab1ce64259**
```
"events": [
        {
          "category": "autoplay",
          "extra": {
            "autoplay_setting": "block_audio"
          },
          "name": "setting_changed"
```

✔️ A user changed their autoplay setting to block audio and video - Ping **24a5ea95-e24b-41ea-b224-384514d1d54a**
```
events": [
        {
          "category": "autoplay",
          "extra": {
            "autoplay_setting": "block_all"
          },
          "name": "setting_changed"
```

[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/aj-t3107001)
[Logcat](https://github.com/mozilla-mobile/fenix/files/5005344/Autoplay.txt)

❌ Not available on neither Fenix Beta 80.0.0-beta-1 & Fenix Beta 80.0.0-beta-2
